app/schema.py

```diff
@@ -156,3 +156,7 @@ class Notes(BaseModel):
     mentor_id: constr(max_length=255)
     mentee_id: constr(max_length=255)
 
```

Comment:
Nice work transferring over more from the Postgres database's tables. Good datatype assignments and Literals. Much to consider as the schema continues to adapt. My only note is to edit the stray spaces out for the checkboxes in the PR description, so markdown renders those better.